qiskit_ibm_provider/job/ibm_composite_job.py

```diff
@@ -752,7 +761,7 @@ def queue_info(self) -> Optional[QueueInfo]:
             self._queue_info = None
         return self._queue_info
 
-    def creation_date(self) -> Optional[datetime]:
+    def creation_date(self) -> Optional[python_datetime]:
```

Comment:
I don't think we have to make any changes to this file because composite jobs are supported anywhere

Reply:
I agree, but if I don't modify the code here, it will fail, so I think this is the cleanest solution, unless we remove all the code for composite jobs.